main.go

```diff
@@ -34,24 +34,33 @@ func main() {
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/" {
```

Comment:
Why is this needed?

Reply:
Actually not, it just means `/foo` will show `/` and that's why we had to do `/swagger.json/`, but removed them, the health endpoint should be enough (tried a few of the suggestions from S/O)